app/models/translation.rb

```diff
@@ -24,8 +24,17 @@ class Translation < ActiveRecord::Base
   # Maps from status symbol to integer representation for database:
   @@STATUS_TO_INT_HASH = @@STATUS_HASH.invert
 
-  def points
-    @@POINTS_HASH[self.status_symbol]
+  def points(since)
```

Comment:
You should probable include a default option, such as `def points(since = 1.month.ago)`.